pkg/R/sparkR.R

```diff
@@ -110,9 +110,19 @@ sparkR.init <- function(
                 mainClass = "edu.berkeley.cs.amplab.sparkr.SparkRBackend",
                 args = as.character(sparkRBackendPort),
                 javaOpts = paste("-Xmx", sparkMem, sep = ""))
-  Sys.sleep(2) # Wait for backend to come up
+
+  cat("Waiting JVM bring up ...\n")
+  while(TRUE) {
+    if(!connExists(.sparkREnv)) {
```

Comment:
@shivaram  We can use `tryCatch` instead, though `.sparkRcon` will be set (for `connExists` to check) once the connection is setup sucessfully.
@felixcheung I think we can set it somewhere else, I just keep it unchanged there as before.